service/src/main/resources/message.properties

```
@@ -0,0 +1,3 @@
+message.not.found = "Entity {0} with id {1} not found"
```

Are you sure that this message means MESSAGE NOT FOUND or it means some entity not found
```suggestion
error.student.not.found = "Student with id {0} not found"
```

---

Done